imap_data_access/cli.py

```diff
@@ -99,13 +104,32 @@ def _query_parser(args: argparse.Namespace):
         "version",
         "extension",
     ]
+    latest_version = ""
     query_params = {
         key: value
         for key, value in vars(args).items()
         if key in valid_args and value is not None
     }
+
+    # removing version from query if it is 'latest', updating flag
+    if args.version == "latest":
+        del query_params["version"]
```

Comment:
@maxine How would I do that? from what I can tell, `io.query()` does not validate if what is passed in is correct. I would assume checking the version would be similar to checking the instrument names passed in.

Reply:
Yeah, you know, you are right about that. However, this isn't really a validation step so much as an additional piece of functionality. I believe query doesn't validate because we assume that any code that is calling it is pre-validating everything already, whereas from the command line human error is more prone. 

I do kind of think all validation should be moved into the query() call, since anyone using the library should get that validation too. But, that should be something we discuss with the team/with Greg since he wrote a lot of this code. 

Since this is a feature add that code might want to use, I do feel it should be in `query` regardless of the other validation. What do you think?

Reply:
I think I get what you are saying